src/cloudEnvs/openLogin.ts

```diff
@@ -4,10 +4,18 @@ import { StateType, TABNINE_URL_QUERY_PARAM } from "../globals/consts";
 import createHubWebView from "../hub/createHubWebView";
 import hubUri from "../hub/hubUri";
 
-export default async function openLogin(): Promise<void> {
+export async function openLogin(): Promise<void> {
```

Comment:
this flow is only used in web based ides (gipod, codespaces) how is it related?

Reply:
because I wanted to poen the signup page which isn't implemented in the binary. in the end not doing that, but need to have the option for later